platforms/blabsy/src/components/input/input.tsx

```
@@ -26,262 +26,265 @@ import type { Tweet } from '@lib/types/tweet';
+            discardTweet();
+            setLoading(false);
+        }
+
+        if (closeModal) closeModal();
+
+        toast.success(
+            () => (
+                <span className='flex gap-2'>
+                    Your Blab was sent
+                    <Link href={`/tweet/${tweetId}`}>
+                        <a className='custom-underline font-bold'>View</a>
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)